libpay-braintree/src/main/scala/com/wix/pay/braintree/BraintreeMerchant.scala

```diff
@@ -1,6 +1,15 @@
 package com.wix.pay.braintree
 
 /**
- * @param merchantAccountIds  Maps currency (ISO 4217) to merchant account id
+ * @param merchantAccountIds  Maps currency (ISO 4217) to merchant account id. on absence default is used (can be set in Braintree account)
  */
-case class BraintreeMerchant(merchantId: String, publicKey: String, privateKey: String, merchantAccountIds: Map[String, String])
+class BraintreeMerchantBase(val merchantAccountIds: Option[Map[String, String]])

```

Comment:
Consider renaming it to "BraintreeMerchant".
Also, what about making this somewhat of a marker interface?
Something like:
trait BraintreeMerchant {
   def merchantAccountIds: Option[Map[String, String]] = None
}